packages/cdktf-cli/bin/cmds/synth.ts

```diff
@@ -37,6 +37,11 @@ class Command implements yargs.CommandModule {
         desc: "Provide JSON output for the generated Terraform configuration.",
         default: false,
       })
+      .option("check-code-maker-output", {
+        type: "boolean",
+        desc: "Whether if check `codeMakerOutput` existence",
```

Comment:
```suggestion
        desc: "Should `codeMakerOutput` existence check be performed? By default it will be checked if providers or modules are configured.",
```